views/default/forms/admin/menu/save.php

```diff
@@ -7,11 +7,7 @@
 $num_featured_items = 6;
 
 // get site menu items
-$menu = elgg_get_config('menus');
-$menu = $menu['site'];
-$builder = new ElggMenuBuilder($menu);
-$menu = $builder->getMenu('name');
-$menu_items = $menu['default'];
+$menu_items = elgg_get_menu('site')->getMenuItems('name', 'default');
```

Comment:
If you follow this code back, you'll see that it comes from ff31226fdeb972aac2f37f0098240cb366a9bb26 which is the original 1.8 admin rewrite from Curverider (pre-new menu system).